### ISSUE TITLE
 Reversed left and right in git gd command

### ISSUE THREAD

**Issue by pouryafard75** [GumTreeDiff/gumtree#338]
Hello, I have started to use the docker image and `git gd` alias which is very cool. 

I think left and right must be reversed for `git gd` command, because based on `git diff`, you can see that I have added a single line of code:

<img width="893" alt="image" src="https://github.com/GumTreeDiff/gumtree/assets/28820932/8d3aac35-659d-485d-a1ce-66b84721b626">

Meanwhile, `git gd` shows the following:
<img width="1424" alt="image" src="https://github.com/GumTreeDiff/gumtree/assets/28820932/38a58707-f04a-47ee-8927-05f8b5addbbf">

basically `/private/$REMOTE` must be mounted to `/diff/right` and `/private/$LOCAL` to `/diff/left` which is surprising according to their names. I will wait for your confirmation in case I am mistaken. 

I can submit the PR as soon as you confirm.

**Comment by jrfaller** [comment]
Hi @pouryafard75 and thanks for the report!

In my gitconfig I have 

```properties
[difftool "gumtree-webdiff"]
cmd = gumtree webdiff "$LOCAL" "$REMOTE"
```

This configuration is working for me.

The README.md for the dockerfile indicates:

```properties
[difftool "gumtree-docker"]
	cmd = docker run -v $REMOTE:/diff/left -v $LOCAL:/diff/right -p 4567:4567 gumtreediff/gumtree webdiff left/ right/
```

Therefore I suspect that indeed $LOCAL should be mapped to left and $REMOTE to right no?

If you have a working configuration I would be grateful for a PR (on the README of the Dockerfile right?) :-)

Cheers!
